sql/core/src/test/scala/org/apache/spark/sql/connector/SupportsCatalogOptionsSuite.scala

```diff
@@ -271,6 +275,47 @@ class SupportsCatalogOptionsSuite extends QueryTest with SharedSparkSession with
     }
   }
 
+  test("mock time travel test") {
+    sql(s"create table $catalogName.t123456789 (id bigint) using $format")
+    sql(s"create table $catalogName.t234567891 (id bigint) using $format")
```

Comment:
I think we need to have a valid timestamp when specifying the time stamp as of value, otherwise, we will get Exception
```
Cannot cast 123456 to TimestampType.
java.time.DateTimeException: Cannot cast 123456 to TimestampType.
	at org.apache.spark.sql.errors.QueryExecutionErrors$.cannotCastUTF8StringToDataTypeError(QueryExecutionErrors.scala:963)
	at org.apache.spark.sql.catalyst.util.DateTimeUtils$.$anonfun$stringToTimestampAnsi$1(DateTimeUtils.scala:431)
	at scala.Option.getOrElse(Option.scala:189)
	at org.apache.spark.sql.catalyst.util.DateTimeUtils$.stringToTimestampAnsi(DateTimeUtils.scala:431)
	at org.apache.spark.sql.connector.expressions.TimeTravelSpec$.create(expressions.scala:378)
```